Tools/scripts/install-travis-env.sh

```diff
@@ -46,60 +34,18 @@ for pkg in $PYTHON_PKGS; do
     sudo pip -q install $pkg || echo "FAILED INSTALL OF $pkg"
 done
 
-# install some extra packages (for later AVR compiler)
-rsync -av firmware.ardupilot.org::Tools/Travis/*.deb ExtraPackages
-sudo dpkg -i ExtraPackages/*.deb || echo "FAILED INSTALL OF EXTRA DEBS"
-
```

Comment:
Yes we need update gcc-avr deb package, please we need grant the server to use rsync, we are getting a conection refusing from server.